raxx_cowboy/lib/raxx/cowboy/handler.ex

```diff
@@ -1,5 +1,16 @@
 defmodule Raxx.Cowboy.Handler do
   @moduledoc false
+
+  def init({:ssl, :http}, req, opts = {router, raxx_options}) do
```

Comment:
ok if those two clauses are the same should we just use
`def init({transport, :http}, req, opts = {router, raxx_options}) when transport in [:tcp, :ssl] do`